daemon/cmd/daemon_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build !privileged_tests
+// +build linux,privileged_tests
```

Comment:
IMO this is unfortunate, would be nice to mock out `bpftool` for this kind of testing if that's the constraining aspect here. I recognize that might not be a small task though.

Reply:
Totally agree. Actually found a way how to avoid marking the suite as privileged and without mocking bpftool: https://github.com/cilium/cilium/pull/11894/commits/d8571facd67a3735eb1117b52e9c173a96a03053.